.circleci/config.yml

```diff
@@ -282,6 +282,7 @@ commands:
             cp ../target/<<parameters.jar_base_name>>.jar .
             cp "$DEPLOY_CONFIG_FILE_DIR/<<parameters.jar_base_name>>.yaml" .
             cp ../output-config/application.conf .
+            cp ../output-config/redisson-jcache.conf .
```

Comment:
New configuration file. There might be a way to consolidate into application.conf but punting on this for now.